### PR TITLE
Added Gopher Gala article.

### PR DESCRIPTION
**Pull request by caser**
Hey - I wrote up a brief article on Gopher Gala. 

I added an image to the top of the article and put it in /public/postimages/gopher-gala/fancy_gopher.jpg 
However, the /public directory is in .gitignore, so the files weren't updated. Wasn't sure exactly why that was, so I left it out - can you please add it in?

Cheers!

fancy_gopher

### CODE REVIEW

**Comment by nathany** [comment]
Hi Casey. That image should go in static. See https://github.com/gopheracademy/gopheracademy-web#working-with-the-blog.

File: upcoming/gopher-gala.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
++++
+author = ["Casey Rosengren"]
+date = "2015-1-3"
+title = "The world's first Golang distributed hackathon."
```
Comment (nathany) [inline upcoming/gopher-gala.md]: Go~~lang~~. likewise below. It's a tag but the name of the language is just "Go"

javascriptlang, rubylang, scalalang. you get the idea.

File: upcoming/gopher-gala.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
++++
+
+# Registration is now open for the Gopher Gala - the world's first Golang distributed hackathon
+
+<center>![](/postimages/gopher-gala/fancy_gopher.jpg "Designed by Renee French")</center>
+
+You may have heard of the Node Knockout or the Rails Rumble. Now, Go has its own distributed hackathon: the [Gopher Gala](http://gophergala.com).
```
Comment (nathany) [inline upcoming/gopher-gala.md]: also the Django Dash :-)

**Comment by nathany** [comment]
Very cool. Hopefully this will bring more participants to the Gala.

**Comment by caser** [comment]
Awesome - made those changes. Let me know if you have any other feedback :)

**Comment by nathany** [comment]
Thanks. Looks good to me.

Hopefully someone comes along to publish it soon. :-)

**Comment by nathany** [comment]
Thanks @ahsanulhaque.
For some reason the image isn't loading: http://blog.gopheracademy.com/gopher-gala/

**Comment by nathany** [comment]
Oh, I think I see why. I missed it in my review. :-(

**Pull request merged**